CONTRIBUTING.md

```diff
@@ -72,3 +72,24 @@ particularly for PRs that affect `IPython.parallel` or Windows.
 
 For more detailed information, see our [GitHub Workflow](https://github.com/ipython/ipython/wiki/Dev:-GitHub-workflow).
 
+## Running Tests
+
+All the tests can by run by using
+```shell
+python3 IPython/testing/iptest.py IPython/core/tests
```

Comment:
I believe you ned to use the iptest entry point that will call iptestcontroller.py otherwise some setup might not be done correctly on all systems. I have to check thoughwhat the exact difference is.

Reply:
On Debian the above command works as expected.